Documentation/diagrams/Architecture.md

```diff
@@ -0,0 +1,322 @@
+# C4 Model
```

Comment:
Let's use lowercase file and folder names and use `docs` instead of `Documentation`.

`docs/diagrams/architecture.md`

Reply:
believe it's done